datadog.go

```diff
@@ -73,6 +74,49 @@ type Options struct {
 	// GlobalTags holds a set of tags that will automatically be applied to all
 	// exported spans.
 	GlobalTags map[string]interface{}
+
+	// DistributionMetricOptions configures distribution metrics
```

Comment:
I'd prefer it if we wouldn't use an embedded structure here. I don't think this is what embedded structures were built for. Let's just add the two options in here instead.

Reply:
sounds good to me